static/js/directives/repo-view/repo-panel-tags.js

```diff
@@ -67,6 +69,34 @@ angular.module('quay').directive('repoPanelTags', function () {
         });
       };
 
+      var matchCosignSignature = function(tag) {
+        // matches cosign style tags and returns the match
+
+        var cosignNamingPattern = new RegExp('^sha256-([A-Fa-f0-9]{64})\.sig$');
```

Comment:
the tag can only be lowercase
```suggestion
        var cosignNamingPattern = new RegExp('^sha256-[0-9a-f]{64}\.sig$');
```